src/ucs/stats/stats.c

```diff
@@ -460,6 +474,171 @@ void ucs_stats_node_free(ucs_stats_node_t *node)
     }
 }
 
+static unsigned
+ucs_stats_agrgt_sum_clsid_alloc(ucs_stats_node_t *node, size_t num_counters_in_class)
```

Comment:
where is the parameter num_counters_in_class being used?

Reply:
Removed.